lib/linguist/languages.yml

```diff
@@ -5088,6 +5080,25 @@ Standard ML:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-ocaml
   language_id: 357
+Starlark:
+  type: programming
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#76d275"
+  extensions:
+  - ".bzl"
```

Comment:
Please also add `".sky"` -- I know you said you didn't want to because it references the old language name, but Bazel (among others) still uses that as the extension for standalone Starlark files. Example: https://github.com/bazelbuild/bazel/tree/master/src/test/starlark/testdata

Reply:
The .star extension is now preferred instead of .sky.

See for example: https://github.com/bazelbuild/starlark/tree/master/test_suite/testdata, https://github.com/google/starlark-go/tree/master/starlark/testdata, https://github.com/drone/drone/issues/2585#issuecomment-501255735, https://github.com/bazelbuild/vscode-bazel/issues/127
I'll rename the test files in the Bazel repository.